README.md

```diff
@@ -1,70 +1,63 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
 
-## Available Scripts
+> My Portfolio
+-It's project built using HTML, CSS3 and react to see how github flow works.
```

Comment:
- [ ] This is not a Portfolio project. We are working on the  Math magicians project. Kindly replace the name with the correct description.